### PR TITLE
#6743 feat: Gallery thumbnail view - image data

### PR DESCRIPTION
**Pull request by creido-welly**
- passed onClick handler from parent Gallery component to child images which are now being passed in as children as opposed to the image data coming in via props
- added image alt text
- image interaction styles

### CODE REVIEW

File: src/components/Gallery/Gallery.tsx [hunk 1]
```diff
@@ -29,29 +31,43 @@ const GalleryMedia = ({
...
   const handleOnClick = (e: any) => {
-    console.log(e.currentTarget);
+    // TODO open gallery lightbox
+    console.log('Gallery handleOnClick', e.currentTarget);
   };
 
+  const childrenWithProps = Children.map(children, child =>
+    cloneElement(child, { onClick: handleOnClick })
+  );
+
```
Comment (chriddell) [inline src/components/Gallery/Gallery.tsx:65]: 👌

File: src/components/Gallery/_gallery.scss [hunk 2]
```diff
@@ -41,17 +41,56 @@
...
+  appearance: none;
+  background: var(--colour-grey-50);
+  border: 0;
+  color: var(--colour-white);
+  cursor: pointer;
+  display: block;
+  height: 100%;
+  outline: 0;
+  position: relative;
+  width: 100%;
```
Comment (chriddell) [inline src/components/Gallery/_gallery.scss:60]: Presuming some of this is resetting `<button>` styles? I'd probably abstract those styles into some kind of btn-reset mixin/extend/util to make it re-usable, but not hugely important

**Review by chriddell — APPROVED** [review]
left a comment re: `button` style abstraction, but shouldn't prevent a merge if you don't want it to!

File: src/components/Gallery/_gallery.scss (hunk 2 above)
Comment (creido-welly) [reply src/components/Gallery/_gallery.scss:60]: indeed :)

**Pull request merged**